OmiseSwift/CurrencyDataConverter.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+
+public class CurrencyDataConverter: Converter {
```

Comment:
I thought about this name at first and then I realized that `Will the API consumer think that this class is a `currency converter` which convert money between currencies?` So I think it should be named with other name